capstone/css/style.css

```diff
@@ -0,0 +1,951 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: 'Lato', sans-serif;
+  margin: 0;
```

Comment:
- [ ] Kindly 😊 try to use the font family of the Behance design to make the font like the Behance fonts, so please try to use these font family types at the fonts of the design Fonts ,so consider using originally: Cocogoose, Lato 😃